src/components/o-commercial-hero-cover/_template.js

```diff
@@ -0,0 +1,26 @@
+import html from 'nanohtml';
+
+export default ({
+                  src,
+                  alt = 'Commercial Hero cover',
+                  gradient = 'white',
+                  contentAlign = 'left'
+}, childrenFragment) => html`
+  <header class="o-commercial-hero-cover__header ${gradient === "white" ? "gradient-white" : "gradient-black"}">
+    <picture class="m-picture o-commercial-hero-cover__picture">
+      <img class="m-picture__image o-commercial-hero-cover__picture__image a-focal-point a-focal-point--center"
+      src="${src}" alt="${alt}" data-object-fit="cover"
```

Comment:
if i remember correctly, this is the polyfill used for IE11 for object fitting.. and if the picture element does not have srcsets maybe the picture tag is not necessary.. but in aem we need it.. 

my suggestion, lets keep the picture tag and add the polyfill as import to this template as well so that the data- is correct. I will make an issue in the plib for creating the picture element

Reply:
this is not only for IE11, it's how we fit all images in AEM.
For most browsers, this works with CSS, but there is a polyfill for IE11.

I think we should simply remove the a-focal-point classes here and add them in AEM.

Reply:
ah, no, this is not going to work... we need these classes.

Reply:
object-fit property passed as an argument and polyfill for object-fit added.
we should be able to reuse the object-fit property once we implement the picture element in the plib